README.md

```diff
@@ -1,2 +1,15 @@
-# InteractiveProgramming
-This is the base repo for the interactive programming project for Software Design, Spring 2016 at Olin College.
+How to Play Anil and Nathan's Pygame!
+
+step1 - make sure you have all the images required to display.
+
+  - They are all in the repo, so as long as it's cloned, you're good to go.
+
+step2 - run "Final.py" in python3
```

Comment:
Even though it's pretty simple, it's good to give the user the actual command to run in the terminal - in this case `python3 Final.py`.